src/Disks/VolumeJBOD.h

```diff
@@ -38,11 +55,18 @@ class VolumeJBOD : public IVolume
     /// Returns valid reservation or nullptr if there is no space left on any disk.
     ReservationPtr reserve(UInt64 bytes) override;
 
+    bool areMergesAllowed() const override;
+
+    void setAllowMergesUserOverride(bool allow) override;
+
+    /// True if parts on this volume participate in merges according to configuration.
+    bool are_merges_allowed = true;
+
+    /// True if parts on this volume participate in merges according to START/STOP MERGES ON VOLUME.
+    std::optional<bool> are_merges_allowed_user_override;
```

Comment:
Why we cannot just modify `are_merges_allowed` on `SYSTEM STOP/START MERGES`?

Reply:
Because this option can come also from configuration and we don't want it to overwrite user settings.